clients/wallet/src/lib.rs

```diff
@@ -12,6 +12,7 @@ pub mod operations;
 mod stellar_wallet;
 mod task;
 pub mod types;
+pub mod keys;
```

Comment:
Let's hide this behind a feature flag then. This should make it obvious that these keys are only used for testing purposes. 
```suggestion
#[cfg(feature = "testing-utils")]
pub mod keys;
```